ttnn/ttnn/__init__.py

```diff
@@ -127,11 +131,43 @@ def save_config_to_json_file(json_path):
         save_config_to_json_file(CONFIG_PATH)
 
 
+def _check__ttnn_so_rpath():
```

Comment:
Please make this an importable function from `tt_lib` instead, with parameters for:

- `so_name` or something similar, for usage in `f.startswith(so_name)`
- `new_path` or something similar

Reply:
ok, I had edit it to make importable